src/utils/index.js

```diff
@@ -1 +1,12 @@
 export const scale = (number, min, max, lowest, highest) => (max-min)*(number-lowest)/(highest-lowest)+min
+
+export const distance = (x1, y1, x2, y2) =>
```

Comment:
We can use "manhattan" or squared distance for closest point comparisons to avoid needing a sqrt call